docs/deploy/installation.md

```diff
@@ -25,26 +22,6 @@ helm install kuberay-operator kuberay/kuberay-operator
 
 #### Method 2: Kustomize
 ```sh
-# Install CRDs
-kubectl create -k "github.com/ray-project/kuberay/manifests/cluster-scope-resources?ref=v1.0.0&timeout=90s"
-
-# Install KubeRay operator
-kubectl apply -k "github.com/ray-project/kuberay/manifests/base?ref=v1.0.0"
-```
-
-> Observe that we must use `kubectl create` to install cluster-scoped resources.
-> The corresponding `kubectl apply` command will not work. See [KubeRay issue #271](https://github.com/ray-project/kuberay/issues/271).
-
-### Single Namespace version
```

Comment:
If users want to use this feature, users should use [Helm chart](https://docs.ray.io/en/latest/cluster/kubernetes/user-guides/helm-chart-rbac.html#kuberay-helm-chart-rbac) instead.